esx_service/vmdk_ops_test.py

```diff
@@ -445,6 +447,18 @@ def remove_vm(si, vm):
         task = vm.Destroy_Task()
         vmdk_ops.wait_for_tasks(si, [task])
 
+def checkIfVolumeExist(volume_names,result):
+    #checks if volume exists in the list of vmdks present in result
+    if result:
```

Comment:
this would be much better (and less nesting) if it said:
```python
if not result:
   return True

for name in volume_names:
    ...   
```
also I do not understnad the login... `if not result` returns true in the code. Why ?